quirks/password-rules.json

```diff
@@ -267,6 +267,9 @@
     "pixnet.cc": {
         "password-rules": "minlength: 4; maxlength: 16; allowed: lower, upper;"
     },
+    "planetary.org": {
+        "password-rules": "required: lower; required: upper; required: digit; minlength: 5; maxlength: 20;"
```

Comment:
Let’s make this comprehensive in terms of allowed symbols.

Reply:
How do I use the `allowed` key? The validation tool doesn't seem to like `allowed: ascii-printable;`

<img width="1025" alt="Screen Shot 2020-07-13 at 8 23 08 PM" src="https://user-images.githubusercontent.com/5855073/87369674-b13a6c00-c546-11ea-92dc-7d41b66a32ce.png">

Reply:
This was a bug in the tool, which we just pushed a fix for. Try again now!